doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -0,0 +1,108 @@
+
+      .. tabs::
+
+         .. group-tab:: west
+
+            1. |open_terminal_window_with_environment|
+            #. Run the following command to erase the flash memory of the VPR core and program the VPR sample:
+
+               .. code-block:: console
+
+                  west flash --erase
+
+            #. Navigate to the build folder of the application sample and run the same command to erase the flash memory of the application core and program the application sample:
+
+               .. code-block:: console
+
+                  west flash --erase
+
+         .. group-tab:: nRF Util
+
+            1. |open_terminal_window_with_environment|
+            #. Run the following command to erase the flash memory of the VPR core and program the VPR sample:
+
+               .. code-block:: console
+
+                  nrfutil device program --firmware zephyr.hex --options chip_erase_mode=ERASE_ALL --core Network
+
+               .. note::
+                    If you cannot locate the build folder of the VPR sample, look for a folder with one of these names inside the build folder of the application sample:
+
+                    * :file:`rpc_host`
+                    * :file:`hci_rpsmg`
+                    * :file:`802154_rpmsg`
+                    * :file:`multiprotocol_rpmsg`
+
+            #. Navigate to the build folder of the application sample and run the following command to erase the flash memory of the application core and program the application sample:
+
+               .. code-block:: console
+
+                  nrfutil device program --firmware zephyr.hex  --options chip_erase_mode=ERASE_ALL
+
+               .. note::
+                    The application build folder will be in a sub-directory which is the name of the folder of the application
+
+            #. Reset the development kit:
+
+               .. code-block:: console
+
+                  nrfutil device reset --reset-kind=RESET_PIN
+
+      See :ref:`readback_protection_error` if you encounter an error.
+
+   .. group-tab:: Sysbuild
+
+      To build and program a sysbuild HEX file, follow the instructions in :ref:`programming_cmd` for the application core sample.
+
+      To program the multi-image HEX file, you can use west or nRF Util.
+
+      .. tabs::
+
+         .. group-tab:: west
+
+            Enter the following command to program multi-image builds for different cores:
+
+            .. code-block:: console
+
+               west flash
+
+            .. note::
+                 The minimum supported version of nrfjprog for multi-image builds for different cores is 10.21.0.
+
+         .. group-tab:: nRF Util
+
+            Enter the following commands to program multiple image builds for different cores:
+
+            .. Commands TBD
+
+            .. code-block:: console
+
+               nrfutil device program --firmware merged_CPUNET.hex --options verify=VERIFY_READ,chip_erase_mode=ERASE_CTRL_AP
```

Comment:
Now I'm confused. This chapter is about nRF54L. There is no Network CPU in nRF54L. This command seems invalid.

Reply:
This was a draft based on nRF53 article, I left the command with the above line saying ".. Commands TBD", as I didn't have an input